acvm_js/build.rs

```diff
@@ -0,0 +1,25 @@
+const GIT_COMMIT: &&str = &"GIT_COMMIT";
+const BARRETENBERG_BIN_DIR: &&str = &"BARRETENBERG_BIN_DIR";
+
+fn main() -> Result<(), String> {
+    if std::env::var(GIT_COMMIT).is_err() {
+        build_data::set_GIT_COMMIT();
+        build_data::set_GIT_DIRTY();
+        build_data::no_debug_rebuilds();
+    }
+
+    match std::env::var(BARRETENBERG_BIN_DIR) {
+        Ok(bindir) => {
+            println!("cargo:rustc-env={BARRETENBERG_BIN_DIR}={bindir}");
+            Ok(())
+        }
```

Comment:
This is a hack so that we can do nix build of barretenberg to $CWD/result and pick that up when org workflow does not allow to define `BARRETENBERG_BIN_DIR`

Reply:
Reverted to nix workflow instead